lib/omnibus/packagers/msi.rb

```diff
@@ -178,6 +185,116 @@ def wix_candle_extension(extension)
     expose :wix_candle_extension
 
     #
+    # Set the name of the certificate store to use when signing packages.
+    #
+    # @example
+    #   cert_store_name "MY"
+    #
+    # @param [String] name
+    #   Name of the certificate store
+    #
+    # @return [String]
+    #   The name of the certificate store to be used
+    #
+    def cert_store_name(name = NULL)
+      if null?(name)
+        @cert_store_name
+      else
+        unless name.is_a?(String)
+          raise InvalidValue.new(:cert_store_name, 'be a String')
+        end
+        @cert_store_name = name
+      end
+    end
+    expose :cert_store_name
+
+    #
+    # Set the name of the certificate use when signing packages.
+    #
+    # @example
+    #   cert_name "MySigningCert"
+    #
+    # @param [String] name
+    #   Name of the certificate
+    #
+    # @return [String]
+    #   The name of the certificate store to be used
+    #
+    def cert_name(name = NULL)
+      if null?(name)
+        @cert_name
+      else
+        unless name.is_a?(String)
+          raise InvalidValue.new(:cert_name, 'be a String')
+        end
+        @cert_name = name
+      end
+    end
+    expose :cert_name
+
+
+    #
+    # Add a timestamp server to be tried.
+    #
+    # Setting nothing will default to trying ['http://timestamp.digicert.com', 
+    # 'http://timestamp.verisign.com/scripts/timestamp.dll']
+    #
+    # If you specify multiple timestamp servers, they will be tried in
+    # the order specified until one of them succeeds
+    #
+    # @example
+    #   timestamp_server "http://timestamp.globalsign.com/scripts/timestamp.dll"
+    #
+    # @param [String] url
+    #   Url of the timestamp server
+    #
+    # @return [Array]
+    #   A list of the timestamp servers to be used
+    #
+    def timestamp_server(url)
+      unless url.is_a?(String)
+        raise InvalidValue.new(:timestamp_server, 'be a String')
+      end
+      @timestamp_servers ||= []
+      @timestamp_servers << url
+      @timestamp_servers
+    end
+    expose :timestamp_server
+
+    #
+    # Set the list of timestamp servers to be tried.
+    #
+    # Setting nothing will default to trying ['http://timestamp.digicert.com', 
+    # 'http://timestamp.verisign.com/scripts/timestamp.dll']
+    #
+    # If you specify multiple timestamp servers, they will be tried in
+    # the order specified until one of them succeeds
+    #
+    # @example
+    #   timestamp_server "http://timestamp.globalsign.com/scripts/timestamp.dll"
+    #
+    # @param [Array] url
+    #   Urls of timestamp servers
+    #
+    # @return [Array]
+    #   A list of the timestamp servers to be used
+    #
+    def timestamp_servers(urls = NULL)
+      if null?(urls)
+        @timestamp_servers || ['http://timestamp.digicert.com',
```

Comment:
:+1: on the constant

cant use ssl. None of the timestamp servers provide that. I'm guessing it is ok because they are signing the hash that is provided with a timestamp, so signtool.exe has enough information to verify it